src/rust/dbv2/migrations/2024-02-15-145709_liquidity/up.sql

```diff
@@ -0,0 +1,60 @@
+-- Your SQL goes here
+CREATE TABLE aggregator.liquidity_groups (
+    group_id SERIAL NOT NULL PRIMARY KEY,
+    "name" TEXT NOT NULL,
+    market_id NUMERIC(20,0) NOT NULL
+);
+
+CREATE TABLE aggregator.liquidity_group_members (
+    group_id INT REFERENCES aggregator.liquidity_groups ("group_id"),
+    "address" TEXT,
+    PRIMARY KEY (group_id, "address")
+);
+
+CREATE TABLE aggregator.liquidity (
+    group_id INT NOT NULL REFERENCES aggregator.liquidity_groups ("group_id"),
+    "time" TIMESTAMPTZ,
+    bps SMALLINT,
```

Comment:
Should be `bps_times_ten` correct?

Otherwise, the intuitive query for 100 bps would actually return a value corresponding to 10 bps